### PR TITLE
Bug/dnn 10523

### PR DESCRIPTION
**Pull request by Mohtshum**
dnn-10523 : Webform postback in Edit mode loosing viewstate of the GridView
- Remove the code updating the update panel in Ids for panels
- Remove the call for EnsureChildControls which will be called by during prerender event by its self during page life cycle event, when viewstate is present.

### CODE REVIEW

File: src/Modules/UI/Dnn.EditBar.UI/Controllers/ContentEditorManager.cs [hunk 1]
```diff
@@ -161,40 +159,11 @@ protected override void CreateChildControls()
                 {
                     continue;
                 }
-
```
Comment (Mohtshum) [inline src/Modules/UI/Dnn.EditBar.UI/Controllers/ContentEditorManager.cs:164]: @zyhfish  please review this change

Comment (zyhfish) [reply src/Modules/UI/Dnn.EditBar.UI/Controllers/ContentEditorManager.cs:164]: These code can't be removed, there have issues related to these changes.

**Review by zyhfish — CHANGES_REQUESTED** [review]

File: src/Modules/UI/Dnn.EditBar.UI/Controllers/ContentEditorManager.cs (hunk 1 above)
Comment (Mohtshum) [reply src/Modules/UI/Dnn.EditBar.UI/Controllers/ContentEditorManager.cs:164]: Hi @zyhfish, those changes seems to be done for https://dnntracker.atlassian.net/browse/CONTENT-5817, which is also running fine after this fix on framework 4.5

**Comment by ohine** [comment]
@Mohtshum can you please resubmit this on the combined repository at https://github.com/dnnsoftware/Dnn.AdminExperience

**Comment by Mohtshum** [comment]
These changes were'nt successfully resolving the issue

**Pull request closed without merging**